sdk/python/endpoints/batch/deploy-pipelines/training-with-components/sdk-deploy-and-test.ipynb

```diff
@@ -1008,11 +1008,19 @@
    },
    "outputs": [],
    "source": [
-    "job = ml_client.batch_endpoints.invoke(\n",
-    "    endpoint_name=endpoint.name,\n",
-    "    deployment_name=deployment_onehot.name,\n",
-    "    inputs={\"input_data\": input_data},\n",
-    ")"
+    "from azure.core.exceptions import ServiceResponseError\n",
+    "\n",
+    "for i in range(3):\n",
+    "    try:\n",
+    "        job = ml_client.batch_endpoints.invoke(\n",
+    "            endpoint_name=endpoint.name,\n",
```

Comment:
Nit: I wonder if there is any retry lib (like Polly) for python.

Reply:
There are a handful of retry libraries. [Tenacity](https://tenacity.readthedocs.io/en/latest/) seems to be the most popular. I would prefer to avoid adding a third party library dependency when all we need is a simple retry. If this simple retry is insufficient, we can discuss using a library.